File: java/Goto.java
class Goto {

	public static void main(String[] args) {
		go_to(1);
	}

	public static void go_to(int n) {
		switch (n) {
		case 1:
			go_to(4);
			break;
		case 2:
			System.out.print("bruh ");
		case 3:
			System.out.print("moment");
			break;
		case 4:
			go_to(2);
		}
	}
}

File: cpp/goto.cpp
#include <iostream>

using namespace std;

int main()
{
	while (true)
	{
		while (true)
		{
			// Break out of both loops
			goto hell;
		}
	}
	hell:
	cout << "Exited loop\n";

	// Can be written using a flag

	bool leaveLoop = false;
	while (true)
	{
		if (leaveLoop) break;
		while (true)
		{
			// Break out of both loops
			leaveLoop = true;
			break;
		}
	}
	cout << "Exited loop\n";

	return 0;
}